go-controller/pkg/libovsdbops/dhcp.go

```diff
@@ -0,0 +1,70 @@
+package libovsdbops
+
+import (
+	libovsdbclient "github.com/ovn-org/libovsdb/client"
+	libovsdb "github.com/ovn-org/libovsdb/ovsdb"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
+)
+
+type dhcpOptionsPredicate func(*nbdb.DHCPOptions) bool
+
+// CreateOrUpdateDhcpOptionsOps will configure logical switch port DHCPv4Options and DHCPv6Options fields with
+// options at dhcpv4Options and dhcpv6Options arguments and create/update DHCPOptions objects that matches the
+// pv4 and pv6 predicates. The DHCP options not provided will be reset to nil the LSP fields.
```

Comment:
Is this what you're trying to say ? I think the following is less confusing.

```suggestion
// pv4 and pv6 predicates. The missing DHCP options will default to nil in the LSP attributes.
```

Maybe "omitted" is better than missing.